controlplane/pkg/apis/connectioncontext/helpers.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net"
 )
 
-func (c *ConnectionContext) IsComplete() error {
+func (c *ConnectionContext) IsValid() error {
```

Comment:
Can we rename this to something not starting with Is? Convention makes it sound like this returns a boolean.

Perhaps Validate() or something similar?

Reply:
Yes, you are right. ÌsValid`should return boolean by name convention.  I'd not changed this method name because we already have similar methods in the project. For example ÌsValid() for `ExtraPrefixRequest`.  Also in the proposal (#811 ) for this issue talking about creation ìsValid()` method. 

Can I also change all similar methods which have a signature `IsValid() error`?